src/style/core.css

```diff
@@ -246,3 +246,17 @@ input:invalid, select:invalid, textarea:invalid, .form-error {
     content: "\002A";
     color: red;
 }
+
+/**
+ * bootstrap tab container mods
+ */
+.zikula-bootstrap-tab-container .tab-content {
+    padding: 1.5em 1em 0 0;
+    border: 1px solid #ddd;
+    border-top: 0;
+    margin-bottom: 1.5em;
+    background-color: #fff;
+}
+.zikula-bootstrap-tab-container .nav-tabs li {
+    margin: 0 0 -1px 0;
+}
```

Comment:
Are these classes intended for common reusage? Should they be utilised by other modules, too?